src/createServer.js

```diff
@@ -1,11 +1,25 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+
+const usersRouter = require('../routes/users');
+const expencesRouter = require('../routes/expences');
```

Comment:
There is a typo in the variable name 'expencesRouter', it should be 'expensesRouter'. Please update the variable name and all its usages.